src/quo2/components/list_items/preview_list.cljs

```diff
@@ -47,11 +54,10 @@
                                      :border-radius border-radius}}]))
 
 (defn list-item
-  [index type size item list-size margin-left
```

Comment:
I'd suggest using a single map argument with keys `:index`, `:type`, `:size`... etc. here. Because with this amount of arguments it becomes harder to maintain they order)

Reply:
@vkjr Again, I agree with your suggestion, but I didn't change this function signature. That's not the scope of this PR :+1: